phpstan.neon.dist

```diff
@@ -64,31 +47,15 @@ parameters:
         - '~^Access to an undefined property atk4\\ui\\AbstractView\:\:\$cb\.$~'
         - '~^Call to an undefined method atk4\\ui\\AbstractView\:\:setModel\(\)\.$~'
         - '~^Call to an undefined method atk4\\ui\\AbstractView\:\:jsReload\(\)\.$~'
-        # for demos/_unit-test/scope-builder-to-query.php
-        - '~^Method atk4\\ui\\App\:\:decodeJson\(\) invoked with 2 parameters, 1 required\.$~'
         # for demos/basic/label.php
         - '~^Call to an undefined method atk4\\ui\\Jquery\:\:fadeOut\(\)\.$~'
```

Comment:
we can fix all `...ndefined method atk4\\ui\\Jquery...` errors easily, just find a list of Jquery functions and phpdoc them

1/2 of the remain error list will be solved